doc/source/tutorial/stats/resampling.rst

```diff
@@ -81,7 +81,7 @@ The demon replies:
     >>> from scipy.stats import binom
     >>> prob = binom.cdf(x, n, p)
     >>> print(f"The correct answer is approximately {prob}")
-    The correct answer is approximately 0.18410080866334788
+    The correct answer is approximately 0.18410080866334808
```

Comment:
The correct value acc. to mpmath is `0.18410080866334813`, so the new value is closer.